CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This can happen when the client creates/updates objects that do not match any su
 * `Realm.App.Sync#pause()` could hold a reference to the database open after shutting down the sync session, preventing users from being able to delete the Realm. ([realm/realm-core#6372](https://github.com/realm/realm-core/issues/6372), since v11.5.0)
 * Fixed a bug that may have resulted in `Realm.Results` and `Realm.List` being in different orders on different devices. Moreover, some cases of the error message `Invalid prior_size` may have be fixed too. ([realm/realm-core#6191](https://github.com/realm/realm-core/issues/6191), since v10.15.0)
 * Exposed `Sync` as named export. [#5649](https://github.com/realm/realm-js/issues/5649)
+* Added missing implementation of `User.state`. [#5686](https://github.com/realm/realm-js/issues/5686)
```

Comment:
Keep under `Fixed`?

Reply:
Yes, it looks more like a bug than an enhancement.